src/main/java/io/vertx/core/eventbus/impl/clustered/Serializer.java

```diff
@@ -73,7 +75,8 @@ private void close(Promise<Void> promise) {
     promise.complete();
   }
 
-  private class SerializerQueue {
+  @SuppressWarnings({"rawtypes", "unchecked"})
+  private class SerializerQueue implements Handler<AsyncResult> {
```

Comment:
what is the type arg for AsyncResult ?

Reply:
It's the same as of `SerializedTask`. But we can't put a wildcard here, the compiler does not accept it.